chimedb/core/util.py

```diff
@@ -1,5 +1,8 @@
 import logging
 
+TIMESTAMP_FORMAT = "%Y-%m-%d-%H:%M:%S.%f"
```

Comment:
What is this for?

Reply:
Also, why is there a dash between the date and the time?  If you want something that isn't a space, use `"T"` per ISO 8601.

Reply:
It is used in several places in comet and the comet archiver. I think it makes sense for it to be up here, so it can be used to interpret this data. If we have something similar somewhere that I could use instead, that would be better.

Changing the colon to a`T` makes sense. But all the dumped data that is waiting to be archived uses the current format.

Reply:
If it's comet-specific, wouldn't it be better to put it somewhere comet-specific?  It's just weird to have it here apropos of nothing.  `chimedb.core` shouldn't have submodule-specific format metadata in it.  Just put it in the `chimedb.comet` package.

This whole file should probably be deleted.  That check_tables function is obsolete.  Richard wrote a new one and put in `orm.py`.

(Also, not the colons, they're fine, but the last dash: `"%Y-%m-%dT%H:%M:%S.%f"`)